.github/workflows/examples.yml

```
@@ -0,0 +1,168 @@
+    branches:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  # Wait for tests from the Tests workflow to complete
+  wait-for-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Wait for Tests workflow to complete
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)